packages/flutter/lib/src/widgets/system_context_menu.dart

```diff
@@ -160,7 +160,14 @@ class SystemContextMenu extends StatefulWidget {
       if (editableTextState.copyEnabled) const IOSSystemContextMenuItemCopy(),
       if (editableTextState.cutEnabled) const IOSSystemContextMenuItemCut(),
       if (editableTextState.pasteEnabled) const IOSSystemContextMenuItemPaste(),
-      if (editableTextState.selectAllEnabled) const IOSSystemContextMenuItemSelectAll(),
+      if (editableTextState.selectAllEnabled)
+        IOSSystemContextMenuItemCustom(
```

Comment:
This could also be made into its own class/ replace the existing select all class so it can be available to more users, something like:

```dart
final class IOSSystemContextMenuItemSelectAll extends IOSSystemContextMenuItem {
  /// Creates an instance of [IOSSystemContextMenuItemSelectAll].
  const IOSSystemContextMenuItemSelectAll({required this.editableTextState});

  final EditableTextState editableTextState;

  @override
  IOSSystemContextMenuItemData getData(WidgetsLocalizations localizations) {
    return IOSSystemContextMenuItemDataCustom(
      title: localizations.selectAllButtonLabel,
      attributes: IOSSystemContextMenuItemAttributes.keepsMenuPresented,
      onPressed: () => editableTextState.selectAll(SelectionChangedCause.toolbar),
    );
  }
}
```